ShellPkg/Library/UefiShellBcfgCommandLib/UefiShellBcfgCommandLib.c

```diff
@@ -202,6 +202,12 @@ GetBootOptionCrc (
                   );
   if (Status == EFI_BUFFER_TOO_SMALL) {
     Buffer = AllocateZeroPool (BufferSize);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (Buffer == NULL) {
+      return (EFI_OUT_OF_RESOURCES);

```

Comment:
Why the parenteses around `EFI_OUT_OF_RESOURCES` here?

Reply:
Some returns have parens like this and some don't throughout ShellPkg. I was trying to keep style consistency within files but because there isn't consistency within this file I can just remove the parens.